### PR TITLE
(Docs): Added GSoC Announcement Blog Post

### PR DESCRIPTION
**Pull request by Abhijay007**
# Summary of Changes

Added GSoC Announcement Blog Post

## Checklist

- [x] My code is well-documented, and I've updated relevant documentation.
- [x] I have tested these changes on my local environment.
- [x] The branch is up-to-date with the base branch.

## Reviewer(s)

@mention any specific reviewers or tag relevant team members.

### CODE REVIEW

File: blog/2024-01-23-GSoC-Annoucement/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+For detailed information about available project ideas, please refer to our [blog](https://uptane.org/blog/2024/01/12/callForIdeas).
+
+**How Do I Apply?** 📝
+
+We urge interested participants to engage with the Uptane community early on, try to join our weekly and bi-weekly Uptane standard meetings, and become active members of our Discord server to stay informed about upcoming events and discussions. Doing this will help you to get a better understanding of the projects and the Uptane community, which will be very beneficial while choosing projects and preparing your proposals.
+
+Although the application period officially commences on March 18th (UTC), we advise candidates to commence preparations without delay. Utilize this time to refine project proposals, familiarize yourselves with Uptane, and delve deeper into your proposed areas of study. Detailed project descriptions are available here on The Linux Foundation's official [idealist](https://wiki.linuxfoundation.org/gsoc/2024-gsoc-uptane), and our mentors are preparing resources to aid contributors in their initial exploration.
```
[the remark this answers is not in the document]
Comment (Abhijay007) [reply blog/2024-01-23-GSoC-Annoucement/index.md:27]: it has the link attached and []() is the syntax for markdown

File: blog/2024-01-23-GSoC-Annoucement/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - Abhij
+---
+
+![](GSoC-Annoucement.png)
+
+Greetings, Uptane community! 👋
+
+We are thrilled to announce that Uptane's sponsoring organization, the Linux Foundation,  has officially become a part of the 2024 edition of Google Summer of Code (GSoC). This will mark Uptane's third year of mentoring GSoC projects under the Linux Foundation umbrella and we couldn't be more excited about the opportunities it presents. 🎉
+
+At its core, GSoC represents more than just a mentoring program; it's a gateway to welcome and engage new contributors to the realm of open source software. It embodies the spirit of giving back, of investing a small portion of time to make a monumental difference, not only in the future endeavors of the GSoC contributors, but also for the broader Uptane community that will reap benefits of these collaborations. 🌟
```
Comment (jhdalek55) [inline blog/2024-01-23-GSoC-Annoucement/index.md:13]:
```suggestion
At its core, GSoC represents more than just a mentoring program; it's a gateway to welcome and engage new contributors to the realm of open source software. It embodies the spirit of giving back, of investing a small portion of time to make a monumental difference, not only in the future endeavors of the GSoC contributors, but also for the broader Uptane community that will reap the benefits of these collaborations. 🌟
```

File: blog/2024-01-23-GSoC-Annoucement/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhdalek55) [reply blog/2024-01-23-GSoC-Annoucement/index.md:27]: I understand about the markdown format. I don't understand the phrase "Linux Foundation official idealist." Shouldn't be Linux Foundation official wiki page? or "project listings"? The word "idealist" here is meaningless.

File: blog/2024-01-23-GSoC-Annoucement/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- November 4: Final date for all GSoC contributors to submit their work product and final evaluation.
+- November 11: Final date for mentors to submit evaluations for GSoC contributor projects with extended deadlines.
+
+For further details, please refer to the [GSoC Timeline](https://developers.google.com/open-source/gsoc/timeline).
+
+## Join the Discussion 🗣️
+
+If you have any doubts and want to discuss something or want resources, don't forget to join our [Discord server](https://discord.gg/SUNJ3gjm9j). We have a dedicated channel for GSoC named [#gsoc](https://discord.gg/jJrCAzTxtp) to answer any questions.
+
+Let's make Uptane GSoC 2024 ☀️ a collaborative success 🚀!
```
Comment (jhdalek55) [inline blog/2024-01-23-GSoC-Annoucement/index.md:50]: I would take this phrase out. You say essentially the same thing two paragraphs down.

**Review by jhdalek55 — DISMISSED** [review]
A few more tweaks

**Review by jhdalek55 — COMMENTED** [review]
Good to go.

**Review by jhdalek55 — APPROVED** [review]
Sorry. Hit the wrong button the last time through.

**Pull request merged**